sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/documentanalysis/DocumentAnalysisAsyncClientTest.java

```diff
@@ -1458,8 +1469,69 @@ public void testClassifyAnalyze(HttpClient httpClient,
                     .setPollInterval(durationTestMode).getSyncPoller();
                 AnalyzeResult analyzeResult = syncPoller.getFinalResult();
                 Assertions.assertNotNull(analyzeResult);
-                Assertions.assertEquals(2, analyzeResult.getDocuments().size());
+                Assertions.assertEquals(3, analyzeResult.getDocuments().size());
+                Assertions.assertEquals(analyzeResult.getModelId(), classifierId);
             }, IRS_1040);
         }
     }
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.formrecognizer.documentanalysis.TestUtils#getTestParameters")
+    public void testFormulaPrebuiltRead(HttpClient httpClient,
+                                        DocumentAnalysisServiceVersion serviceVersion) {
+        client = getDocumentAnalysisAsyncClient(httpClient, serviceVersion);
+        testingContainerUrlRunner((sourceUrl) -> {
+            SyncPoller<OperationResult, AnalyzeResult> syncPoller
+                = client.beginAnalyzeDocumentFromUrl("prebuilt-read", sourceUrl,
+                    new AnalyzeDocumentOptions().setDocumentAnalysisFeatures(Collections.singletonList(
+                    DocumentAnalysisFeature.OCR_FORMULA)))
+                .setPollInterval(durationTestMode)
+                .getSyncPoller();
+            syncPoller.waitForCompletion();
+            AnalyzeResult analyzeResult = syncPoller.getFinalResult();
+            Assertions.assertNotNull(analyzeResult.getPages());
+            DocumentFormula formula =
+                analyzeResult.getPages().get(0).getFormulas().get(0);
+            Assertions.assertEquals(DocumentFormulaKind.INLINE, formula.getKind());
+            Assertions.assertEquals("a + b = c .", formula.getValue());
```

Comment:
Just a heads up, we removed this check in Python since it seems the service doesnt always return exactly this value.